cfme/tests/infrastructure/test_customization_template.py

```diff
@@ -63,7 +65,7 @@ def test_pxe_image_type_required_error_validation(collection):
             script_data='Testing the script')
 
 
-@pytest.mark.uncollectif(lambda: version.current_version() < '5.9')
+@pytest.mark.uncollectif(BZ(1449116, forced_streams=['5.7', '5.8']).blocks, reason='BZ 1449116')
```

Comment:
**REQUIRED**: This introduces a module-import-time external access. Please make it a proper blocker or put it into lambda.